db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/CreateIndexNode.java

```diff
@@ -64,8 +68,9 @@
 	String				indexType;
 	TableName			indexName;
 	TableName			tableName;
-	Vector				columnNameList;
-	String[]			columnNames = null;
+	Vector<IndexExpression> expressionList;
```

Comment:
maybe align like other members? (also below)

Reply:
Yes, fixed in the following commits.